lib/restforce/middleware/authentication/jwt_bearer.rb

```diff
@@ -3,32 +3,36 @@
 require 'jwt'
 
 module Restforce
-  class Middleware::Authentication::JWTBearer < Restforce::Middleware::Authentication
-    def params
-      {
-        grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
-        assertion: jwt_bearer_token
-      }
-    end
+  class Middleware
+    class Authentication
+      class JWTBearer < Restforce::Middleware::Authentication
```

Comment:
Not sure how I feel about this, the gem tends to prefer the compact style of `Middleware::Authentication::JWTBearer`, let me know if you'd prefer we tweak the rubocop config?

Would also be good to get the rubocop failures to fail PR checks on github